iocage/lib/Prompts.py

```diff
@@ -0,0 +1,20 @@
+import helpers
+
+
+class Prompts:
+
+    def __init__(self, host=None):
+        helpers.init_host(self, host)
+
+    @property
+    def release(self):
+        i = 0
+        default = None
+        for available_release in self.host.distribution.releases:
+            if available_release.name == self.host.release_version:
+                default = i
+                print(f"[{i}] \033[1m{available_release.name}\033[0m")
```

Comment:
> Printing is bad in libs :)

@skarekrow Of course you are right! Printing will be disabled by default, so that the CLI must manually enable them when using the library.